CloudCulator/Cloudculator.Core.UnitTests/CalculatorEngineUnitTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Cloudculator.Core.UnitTests
+{
+    public class CalculatorEngineUnitTests
+    {
+        [Fact]
+        public void Add_TwoNumbers_ReturnAddition()
+        {
+            //Arrange
+            var service = new CalculatorEngine();
+            double leftArgument = 5;
+            double rightArgument = 10.5;
+            double expectedResult = 15.5;
+
+            //Act
+            var result = service.Add(leftArgument, rightArgument);
+
+            //Assert
+            Assert.Equal(expectedResult, result);
+        }
+
+        [Fact]
+        public void Add_AdditionOfArgumentsOverflowsDoubleMaxPrecision_ThrowsOverflowException()
```

Comment:
What will happen for the following cases:
double.PositiveInfinity
double.NegativeInfinity
double.NaN
double.MaxValue

for either one or both arguments in any combinations?